.travis.yml

```diff
@@ -1,14 +1,18 @@
 language: php
 
-dist: trusty
+dist: xenial
+
+services:
+  - mysql
 
 env:
   global:
-    - COMPOSER_ROOT_VERSION="2.4.x-dev" DB=MYSQL
+    - COMPOSER_ROOT_VERSION="2.x-dev" DB=MYSQL
 
 php:
   - 7.1
   - 7.2
+  - 7.3
```

Comment:
This may not work yet, unless LDAP has been updated to support it

Reply:
just wanted to try :D
who knows...

Reply:
7.3 is green now. Looks like we don't use LDAP in this recipe :-P